### PR TITLE
Website (mobile): Add source code links, remove generic library link

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Each control now has a link to its source code on GitHub. The generic links to the library were less useful.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9487)

### CODE REVIEW

File: apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerImplementation.md [hunk 1]
```diff
@@ -16,8 +16,10 @@
 
 `Widget.UIFabric.CircularProgress.Large.Primary`
 
-### Library
+### Source code
+
+[CiruclarProgress styles](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/values/styles.xml)
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerImplementation.md]:
```suggestion
[CircularProgress styles](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/values/styles.xml)
```

File: apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerImplementation.md [hunk 2]
```diff
@@ -16,8 +16,10 @@
...
 
-[UI Fabric Android](https://github.com/OfficeDev/ui-fabric-android)
+### Sample code
 
-[Sample Code](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_progress.xml)
+[Progress demo xml](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_progress.xml)
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Controls/SpinnerPage/docs/android/SpinnerImplementation.md]:
```suggestion
[Progress demo XML](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_progress.xml)
```

File: apps/fabric-website/src/pages/Controls/SeparatorPage/docs/android/SeparatorImplementation.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
...
-### Library
+### Source code
 
-[UI Fabric Android](https://github.com/OfficeDev/ui-fabric-android)
+[ms\_row\_divider](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/drawable/ms_row_divider.xml)
 
-[Sample Code](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/java/com/microsoft/officeuifabricdemo/DemoListActivity.kt)
+### Sample code
+
+[ms\_row\_divider in xml](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/layout/view_date_time_picker.xml)
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Controls/SeparatorPage/docs/android/SeparatorImplementation.md]:
```suggestion
[ms\_row\_divider in XML](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/layout/view_date_time_picker.xml)
```

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/android/ButtonImplementation.md [hunk 4]
```diff
@@ -4,8 +4,10 @@
...
-### Library
+### Source code
 
-[UI Fabric Android](https://github.com/OfficeDev/ui-fabric-android)
+[Button styles](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/values/styles.xml)
 
-[Sample Code](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_basic_inputs.xml)
+### Sample code
+
+[Button demo xml](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_basic_inputs.xml)
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/android/ButtonImplementation.md]:
```suggestion
[Button demo XML](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_basic_inputs.xml)
```

File: apps/fabric-website/src/pages/Controls/TextPage/docs/android/TextImplementation.md [hunk 5]
```diff
@@ -18,10 +18,12 @@
...
-[UI Fabric Android](https://github.com/OfficeDev/ui-fabric-android)
+[TextAppearance styles](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/values/styles_font.xml)
 
-[Sample Code in Activity](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/java/com/microsoft/officeuifabricdemo/demos/TypographyActivity.kt)
+### Sample code
 
-[Sample Code in Layout](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_typography.xml)
+[TextAppearance in activity](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/java/com/microsoft/officeuifabricdemo/demos/TypographyActivity.kt)
+
+[TextAppearance in xml](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_typography.xml)
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Controls/TextPage/docs/android/TextImplementation.md]:
```suggestion
[TextAppearance in XML](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric.Demo/src/main/res/layout/activity_typography.xml)
```

**Comment by vladfilyakov** [comment]
Source code
MSSegmentedContrl - missing "o"

File: apps/fabric-website/src/pages/Controls/PivotPage/docs/ios/PivotImplementation.md [hunk 6]
```diff
@@ -2,8 +2,10 @@
 
 `MSSegmentedControl`
 
-### Library
+### Source code
 
-[UI Fabric iOS](https://github.com/OfficeDev/ui-fabric-ios)
+[MSSegmentedContrl](https://github.com/OfficeDev/ui-fabric-ios/blob/master/OfficeUIFabric/Controls/MSSegmentedControl.swift)
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/PivotPage/docs/ios/PivotImplementation.md]: For Vlad's comment:
```suggestion
[MSSegmentedControl](https://github.com/OfficeDev/ui-fabric-ios/blob/master/OfficeUIFabric/Controls/MSSegmentedControl.swift)
```

**Comment by vladfilyakov** [comment]
Emily, I was thinking about the text we use to "name" links to demos for both Android and iOS. What do you think of the following:

Android:
<control name> demo activity (added "demo" to align with "demo XML" and to add more clarity)
<control name> demo XML (no changes)

iOS:
<control name> demo (removed controller because it's always a controller + some supporting classes and to make text shorter; used explicit control name so that things like "MSPopupMenu demo controller" become "MSPopupMenuController demo" - control is called MSPopupMenuController)

**Pull request merged**